### PR TITLE
create _package.py for storing package name info

### PR DESCRIPTION
**Pull request by abelnation**
This is the start of some clean-up/organization I'd like to do within the codebase.  I think the project could benefit greatly by mirroring the hierarchy of the three.js code base.  

Anyways, this is just a tiny little PR to gauge whether you're welcoming to contribution or not.  My plan is to split the python libraries out into separate files corresponding to the different sections of the three.js docs:

- cameras
- extras
- materials
- objects
- renderers
- scenes
- textures

etc. ...

The result would be that `__init__.py` imports vars from a number of files as opposed to a single python file. 

I'm curious what you think, and if you're open to me working on some clean-up/organization.  Thanks!

### CODE REVIEW

**Comment by jasongrout** [comment]
We're definitely open to some help refactoring and cleaning up the code! Thanks! I'll try to take a look at this tomorrow (feel free to ping if that doesn't happen, though!)

File: pythreejs/pythreejs.py [hunk 1]
```diff
@@ -15,8 +15,11 @@
 from ipywidgets import Widget, DOMWidget, widget_serialization, Color
 from traitlets import (Unicode, Int, CInt, Instance, Enum, List, Dict, Float,
                        CFloat, Bool)
+from ._package import npm_pkg_name
 from math import pi, sqrt
 
+from core import *
```
Comment (jasongrout) [inline pythreejs/pythreejs.py]: This line isn't doing anything - can we delete it?

**Comment by abelnation** [comment]
@jasongrout what is the best way to discuss ideas/plans with folks?  I have a lot of thoughts on this repo (and was even considering writing my own from scratch, but have decided against that after careful consideration).  I'd like to 1) help in whatever way i can and 2) avoid wasting time implementing ideas the maintainers do not support :)

File: pythreejs/pythreejs.py (hunk 1 above)
Comment (abelnation) [inline pythreejs/pythreejs.py]: yup, done

**Comment by jasongrout** [comment]
Perhaps a conference video-chat with @SylvainCorlay and myself would be the best way forward.

**Comment by jasongrout** [comment]
What times (UTC) are good for you?

**Comment by SylvainCorlay** [comment]
Thanks for the contribution. We are totally open to contributions to pythreejs.

**Comment by abelnation** [comment]
I'm up for a video chat.  I'm PST, and free most of the day.  So, that'd be: 
4PM - 6PM (UTC)
10PM - Midnight (UTC)

**Comment by jasongrout** [comment]
@abelnation - I realize it's not in the time you suggested, but @sylvainCorlay and I could talk now, or we could find a time on Monday.

**Comment by abelnation** [comment]
@jasongrout I could chat right now for a little bit.  hangouts?

**Comment by jasongrout** [comment]
bluejeans.com/jupyter

**Comment by jasongrout** [comment]
Looks great; thanks!

**Pull request merged**